modules/Core/src/main/java/org/terasology/logic/health/HealthComponent.java

```diff
@@ -21,7 +21,7 @@
 
 /**
  */
-public final class HealthComponent implements Component {
```

Comment:
Hmm, making this non-final could have a performance issue...  Java needs a way to use the same memory as a different type as is easy in C++, or to not use individual components, hmm...